.github/workflows/msquic_kernel_bvt.yml

```diff
@@ -0,0 +1,165 @@
+name: MsQuic Kernel BVT
```

Comment:
I am not familiar with netperf, but what is below seems largely duplicated from custom.yml.
Why is it needed twice?

As a general PR feedback, I think some comments explaining the general idea would be useful, especially since we are doing something somewhat out of scope of netperf only to leverage its pool.

Reply:
Custom.yml is just for testing

Reply:
Good to know. IMO, that is worth a comment explaining it at the top of the file, since it is a pretty unusual pattern to check in test code like this.

Reply:
custom.yml is vastly different between branches. It's really not meant to be maintaned. This was a hacky workaround Nick B added which allowed u to run arbitrary workflow yamls from topic branches and having the GH UI update correctly. But thanks for the suggestion!